src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Definition/RazorDefinitionEndpoint.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Threading;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Server;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using Microsoft.AspNetCore.Razor.LanguageServer.ProjectSystem;
+using Microsoft.CodeAnalysis.Razor;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.LanguageServer.Common;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Microsoft.AspNetCore.Razor.Language.Legacy;
+using Microsoft.CodeAnalysis.Razor.ProjectSystem;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Definition
+{
+    class RazorDefinitionEndpoint : IDefinitionHandler
```

Comment:
```suggestion
    internal class RazorDefinitionEndpoint : IDefinitionHandler
```

Reply:
I know you didn't define `IDefinitionHandler` but I don't care for that name. Feel like it ought to be `IGoToDefinition` or some such.